modules/rosa-sdpolicy-platform.adoc

```diff
@@ -0,0 +1,86 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-service-definition.adoc
+
+[id="rosa-sdpolicy-platform_{context}"]
+= Platform
+
+== Cluster backup policy
+
+[IMPORTANT]
+====
+It is critical that customers have a backup plan for their applications and application data.
+====
+Application and application data backups are not a part of the {product-title} service.
+All Kubernetes objects and PVs in each {product-title} cluster are backed up to facilitate a prompt recovery in the unlikely event that a cluster becomes irreparably inoperable.
+
+The backups are stored in a secure object storage (Multi Availability Zone) bucket in the same account as the cluster.
+Node root volumes are not backed up as Red Hat Enterprise Linux CoreOS is fully managed by the OpenShift Container Platform cluster and no stateful data should be stored on a node's root volume.
```

Comment:
s/backed up as/backed up, as
s/Red Hat Enterprise Linux CoreOS/{op-system-first} or "Red Hat CoreOS" if the attribute can't be used in this repo.

Reply:
Done.